examples/demo/pictures/pictures.kv

```diff
@@ -40,8 +40,9 @@ FloatLayout:
         id: image
         source: root.source
 
-        # create initial image to be 400 pixels width
-        size: 400, 400 / self.image_ratio
+        # create initial image with relative size
+        size: win.Window.width * 0.6, win.Window.width * 0.6 / self.image_ratio
```

Comment:
I don't think you need to divide by `self.image_ratio`, because `win.Window.width` should already have then that.

Reply:
I see. Currently, i am trying to get familiar with kivy, so i surfed through the open issues, and found the 'easy' tagged ones. Lacking experience, I intended to change the least possible amount, so i only replaced the 400px of the loaded image by a runtime value, the root's window size.

But i don't understand what `win.Window.width` could do to determine the ratio of the loaded image. As far as i understood the code, `self.image_ratio` is the aspect ratio of the individually loaded image file. The original script, and my proposed change yield such a result:

![pictures_w_ratio](https://user-images.githubusercontent.com/42077282/67165026-c7811600-f380-11e9-8e09-308fa845226f.png)


Using `size: win.Window.width * 0.6, win.Window.width * 0.6` will yield unnecessary white space in the picture's frame. See below: 

![pictures_wo_ratio](https://user-images.githubusercontent.com/42077282/67164983-4c1f6480-f380-11e9-8bf9-fde95722aa8b.png)


Therefore, i disagree. The aspect ratio of the image `self.image_ratio` should be in the computations determining the individual frame size.  Whether, the ratio divides or multiplies the base size is arguable. Of course it is also arguable which base size (`height` or `width`) is to be used as well its ratio (0.6). The proposed choice seemed to give the best impression in landscape and portrait mode.